mne/time_frequency/tfr.py

```diff
@@ -2852,6 +2902,15 @@ def __getstate__(self):
 
     def __setstate__(self, state):
         """Unpack AverageTFR from serialized format."""
+        if state["data"].ndim not in [3, 4]:
+            raise ValueError(
+                f"RawTFR data should be 3D or 4D, got {state['data'].ndim}."
+            )
+        # Set dims now since optional tapers makes it difficult to disentangle later
+        state["dims"] = ("channel",)
+        if state["data"].ndim == 4:
+            state["dims"] += ("taper",)
+        state["dims"] += ("freq", "time")
```

Comment:
Example of handling dims in the `AverageTFR` class where only one dimension (`taper`) is optional.